debian/changelog

```diff
@@ -1,3 +1,38 @@
+ubuntu-advantage-tools (27.7~22.04.1) jammy; urgency=medium
+
+  * d/logrotate:
+    - make new logs world readable
+  * d/tools.postinst:
+    - refactor to catch exception from entitlement_factory
```

Comment:
Not entirely sure if this entry should be in the changelog, since it seems like a change that will only matter to users if they have knowledge about the code.

Reply:
Although the details may be, I'd keep it in the changelog because it is a change to postinst, and reviewers will care about it

Reply:
I see your point, but I think the changelog should be more user focus then reviewer related. But if we want to keep it, no problem

Reply:
iirc, it is mandatory to have the debian/[files] changes in the changelog - if that does not hold, then we should remove it for sure

Reply:
I'm on the fence, but I think I'm leaning towards keeping it. If a user runs into an issue trying to install `ua`, they may look at the changelog and see that maybe the refactor was the problem